test/framework/resources/templates/logger-app.yaml.tpl

```diff
@@ -18,6 +18,8 @@ spec:
     spec:
       containers:
       - name: logger
+        # A custom agnhost image (eu.gcr.io/gardener-project/3rd/agnhost) is used instead of the upsteam one (registry.k8s.io/e2e-test-images/agnhost)
```

Comment:
```suggestion
        # A custom agnhost image (eu.gcr.io/gardener-project/3rd/agnhost) is used instead of the upstream one (registry.k8s.io/e2e-test-images/agnhost)
```